editor/components/provider/index.js

```diff
@@ -52,6 +52,9 @@ class EditorProvider extends Component {
 		// Assume that we don't need to initialize in the case of an error recovery.
 		if ( ! props.recovery ) {
 			this.store.dispatch( setupEditor( props.post, this.settings ) );
+			if ( props.autosave ) {
```

Comment:
Is there ever a case where `props.recovery && props.autosave` ? Do we need to nest this?

Reply:
Probably not, un-nested in https://github.com/WordPress/gutenberg/pull/4218/commits/1ee973e2e2d36f4e550d5fac9582ff1c71fc03ea